mlflow/server/handlers.py

```diff
@@ -588,6 +588,11 @@ def _update_model_version():
     model_version = _get_model_registry_store().update_model_version(
         name=request_message.name, version=request_message.version,
         description=new_description)
+    if request_message.HasField("stage"):
```

Comment:
@tomasatdatabricks This field was deprecated but not removed from update, so I added it back.